api/MediaSource.json

```diff
@@ -669,12 +669,26 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaSource/sourceclose_event",
           "spec_url": "https://w3c.github.io/media-source/#dfn-sourceclose",
           "support": {
-            "chrome": {
-              "version_added": "53"
-            },
-            "chrome_android": {
-              "version_added": "33"
-            },
+            "chrome": [
+              {
+                "version_added": "53"
+              },
+              {
+                "version_added": "31",
```

Comment:
@Elchi3 did you deliberate pick 31 instead of 23 here? And should there be a "version_removed" here too?

Reply:
Ah yes, I think I picked the wrong parent version number. We should assume the event was there with the prefixed version already.

Reply:
Fixed!